### ISSUE TITLE
Writing language preference 

### ISSUE THREAD

**Issue by hkujy** [znck/grammarly#419]
Hi,
In my account, I set using British English. But in the VS code grammerly, it is not. 
thanks

**Comment by pvanheus** [comment]
Yep, I have the same problem. I set "Document Dialect" to `british` but it still says that "analyse" should be "analyze". I now notice that this problem has been reported as #399, #378 and #362 and seems to have started some time after release 0.22.

**Comment by viktaur** [comment]
Same issue here.

**Comment by drvolpe** [comment]
Confirm, I haven't found a way to use British English with this extension, it ignores the setting.
